tests/main_test.py

```diff
@@ -106,16 +114,18 @@ def test_cli_output():
     assert "MD5\n" in result.output
 
 
-def test_cli_no_hashes():
-    runner = CliRunner()
-    result = runner.invoke(main, [])
-    assert result.exit_code == 0
-    assert "Error." in result.output
-
-
 def test_sth_api_key():
     hashes = ["5d41402abc4b2a76b9719d911017c592"]
 
     x = api.return_as_fast_json(hashes, "meow")
 
     assert x is not None
+
+
+def test_cli_fail_on_grep():
```

Comment:
Link to the relevant github issue in a comment

Reply:
Done